Cargo.toml

```diff
@@ -52,15 +55,18 @@ once_cell = "1.18.0"
 owo-colors = { version = "3.5.0", features = ["supports-colors"] }
 path-absolutize = "3.1.1"
 pathdiff = { version = "0.2.1", features = ["camino"] }
+ratatui = "0.24.0"
 regex = { version = "1.9.3", default-features = false, features = ["perf", "std"] }
 shell-words = "1.1.0"
 strip-ansi-escapes = "0.2.0"
 tap = "1.0.1"
+tempfile = "3.8.1"
```

Comment:
`gag` uses a temporary file to redirect `stdout` and `stderr`. Apparently it's guaranteed to be in memory with modern Linux?

I need `tempfile` for the `AsyncBufferRedirect` thing.